velox/functions/prestosql/aggregates/tests/AverageAggregationTest.cpp

```diff
@@ -343,6 +346,23 @@ TEST_F(AverageAggregationTest, avgDecimal) {
       {makeRowVector({makeShortDecimalFlatVector({3'000}, DECIMAL(10, 1))})});
 }
 
+TEST_F(AverageAggregationTest, avgDecimalWithGroupingKeys) {
+  auto input = {
+      makeRowVector(
+          {makeNullableFlatVector<int32_t>({1, 1}),
+           makeShortDecimalFlatVector({37220, 53450}, DECIMAL(5,2))}),
+      makeRowVector(
+          {makeNullableFlatVector<int32_t>({1, 1}),
```

Comment:
May be add a different value for c0. Right now, it just creates 1 group. With more groups (3 or 4), the memory offset and allocation calls will get tested as well.

Reply:
Thanks. I added another test for multiple groups below.

Reply:
Thanks.
Nit: Can you just merge avgDecimalWithMultipleVectors with `avgDecimal` test.  Also, merge grouping tests together. You can add comments on each test.

Reply:
Done, thanks.